record-builder-processor/src/main/java/io/soabase/recordbuilder/processor/InternalRecordBuilderProcessor.java

```diff
@@ -363,7 +363,7 @@ public static MyRecord(int p1, T p2, ...) {
             }
          */
         CodeBlock codeBlock = buildCodeBlock();
-        var builder = MethodSpec.methodBuilder(recordClassType.name())
+        var builder = MethodSpec.methodBuilder(recordClassType.name() + metaData.staticBuilderSuffix())
```

Comment:
```suggestion
           var staticBuilderName = Optional.of(metaData.staticBuilderSuffix()).filter(e -> !e.isEmpty()).map(e -> lowerCaseFirstLetter(recordClassType.name())+ e).orElse(recordClassType.name());
        var builder = MethodSpec.methodBuilder(staticBuilderName)
```

```
   private String lowerCaseFirstLetter(String value) {
        return value.substring(0,1).toLowerCase() + value.substring(1);
    }
```


Might be nicer to lowercase the first letter of the static builder as well so instead of `FirstnameOf` you have `firstnameOf` for example